sqlglot/dialects/prql.py

```diff
@@ -104,8 +104,18 @@ def _parse_take(self, query: exp.Query) -> t.Optional[exp.Query]:
             num = self._parse_number()  # TODO: TAKE for ranges a..b
             return query.limit(num) if num else None
 
+        def _parse_ordered(
+            self, parse_method: t.Optional[t.Callable] = None
+        ) -> t.Optional[exp.Ordered]:
+            asc = self._match(TokenType.PLUS)
+            desc = self._match(TokenType.DASH) or (asc and False)
+            term = super()._parse_ordered(parse_method)
+            if term and desc:
+                return term.desc()
+            return term
+
         def _parse_order_by(self, query: exp.Select) -> t.Optional[exp.Query]:
-            l_brace = self._match(TokenType.L_BRACE)
+            l_brace = self._match(TokenType.L_BRACE)  # TODO: support sort with expression
```

Comment:
It's not very clear what expressions you're referring to here. We do support expressions, just not s-strings yet. No need to have these TODOs imo since this is still in an experimental stage.

```suggestion
            l_brace = self._match(TokenType.L_BRACE)
```